components/Itemlist.js

```diff
@@ -0,0 +1,74 @@
+import { shoes } from "../public/shoes";
+import styled from "styled-components";
+
+export default function Itemlist() {
+  return (
+    <>
```

Comment:
all your Listitems should be in a List (see above) and therefore, this should be a Listitem <li> 
This is actually a single Card, right? - check the naming.